graduation-project-tips.md

```diff
@@ -0,0 +1,93 @@
+- **Do** *knowledge exchange sessions* with each other every now and then to sync everyone's knowledge especially if you're organized as sub-teams.
+
+## Tip #6: Document Everything (literally) :pencil2:
+
+You'll need to document everything that happens in the project life-time. It's a one-year project, don't expect to never forget stuff, on the contrary, you'll always forget details especially finer ones.
+What to document ?
+
+- Requirements Analysis, and project-planning details.
+- Supervisor notes in each meeting with him/her (Meeting Notes).
+- How did you reach an agreement on something (Brainstorming sessions).
+- Links/resources you find along the way (Bugs do re-appear).
+
+## Tip #7: Manage Code Smartly :octocat:
+
+If you didn't use Git (or any other version control system) and Github (or any other remote repositories system) till your senior year for any reason, now is the time. I think I can say that it's a must to manage the project's code using a version control tool as Git and a tool for remote hosting as Github (my favorite options although others exist).
+
+Another tip here, is to use a clean **git branching model** to work with (a lot exist, google that for a specific one). A git branching model is some rules to follow when naming branches, commits and pull requests. It's an important matter to follow to avoid miscommunication and naming in-consistencies.
+
+Last tip here, is to agree on a standard code naming convention to use from the beginning. Naming in-consistencies in the code itself can be really highly confusing and takes time to refactor if left unattended. Save yourselves both time and effort and agree on a coding style guidelines (for each language, there will be couple of options to choose from) to follow from start. 
+
+## Tip #8: Use Your Academic Email :office:
+
+Everyone forgets the precious treasure lying in their hands until they lose it. If you have an academic email, you get free access to a lot of software tools that will be very beneficial in your graduation project. 
+The biggest gain comes from the awesome **[Github Student Pack](https://education.github.com/pack)**, just check the amount of tools you get access to or trials at least (From free tutorials/courses to AWS credits to perform learning/computations on machines).
+If any of the team members don't have such an email, I highly advise him.her to go get one asap because you'll want to use it.
+
+
+## Tip #9: Always Think Big ! :dart:
+
+Take this project seriously, it's going to (should) be the biggest project in your career so far, it will be the center of traction everyone will ask you about. If I am recruiting fresh graduates, that would be my first point to ask about technically.
+Don't just say this is another college project, because you may not get the chance to work on such collaborative project where you build everything from scratch again.
```

Comment:
No need to negative again and assume the reader is negligent :) Also may not necessarily be the biggest project in their career. Let's just focus on how big an opportunity it is without making a lot of strong statements.